SparseGrids/Benchmarks/benchFFT.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef _TASMANIAN_BENCHMARK_FFT_HPP
+#define _TASMANIAN_BENCHMARK_FFT_HPP
+
+#include "benchCommon.hpp"
+
+bool benchmark_fft(std::deque<std::string> &args){
+    if (args.size() < 5) return false;
+
+    // report the test parameters to reference later
+    cout << "fourier";
```

Comment:
Only Global grids have a trivial algorithm for estimating interpolation coefficients, this would be useful for grids other than Fourier. The functions to read and process the command line inputs should be there already (see the other benchmarks). You need to add grid-family, rule and order to the list of cli arguments.